app/ante/evm/eth.go

```diff
@@ -136,7 +136,7 @@ func (egcd EthGasConsumeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simula
 	if ctx.IsReCheckTx() {
 		// Use new context with gasWanted = 0
 		// Otherwise, there's an error on txmempool.postCheck (tendermint)
-		// that is not bubbled up. Thus, the Tx never runs on DeliverMode
+		// that is not bubbled up. Thus, the Tx never runs on deliverymode
```

Comment:
This changes the meaning. `DeliverMode` means "TX on deliver mode", and `deliverymode` assumes "mode of the TX".